html/manifest/basic_support.json

```diff
@@ -0,0 +1,57 @@
+{
+  "progressivewebapps": {
+    "manifest": {
+      "basic_support": {
```

Comment:
This file isn't needed. We've removed the concept of "Basic support" from MDN as it was too vague and we're instead listing all sub features (all manifest keys in this case).